test/console_TEST.cc

```diff
@@ -1,3 +1,37 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2020, Open Source Robotics Foundation, Inc.
```

Comment:
I think this could be 2016 since it's when the file was created.

Reply:
I used 2020 because its current state reflects mostly the work done on the latest two commits made this year, but I can change it if someone else agrees with you.

Reply:
I think either is fine